src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -757,14 +777,15 @@ ucs_status_t uct_dc_mlx5_ep_qp_to_err(uct_dc_mlx5_ep_t *ep)
         return status;
     }
 
+    uct_dc_mlx5_set_dgram_seg(txwq, iface, &ep->av, uct_dc_mlx5_ep_get_grh(ep));
```

Comment:
move this after the comment, right before line 784

Reply:
> move this after the comment, right before line 784

moved